internal/pkg/describe/service.go

```diff
@@ -202,6 +204,22 @@ func NewServiceDescriber(opt NewServiceConfig) (*ServiceDescriber, error) {
 	}, nil
 }
 
+// Platform returns the platform of the task definition. If it is nil, fill in default values.
+func (d *ServiceDescriber) Platform() (*awsecs.ContainerPlatform, error) {
+	taskDefinition, err := d.ecsClient.TaskDefinition(d.app, d.env, d.service)
+	if err != nil {
+		return nil, fmt.Errorf("describe task definition for service %s: %w", d.service, err)
+	}
+	platform := taskDefinition.Platform()
```

Comment:
Would the sdk return an empty `platform` if it is `linux/x86_64`? Do you know at what conditions would `platform == nil` happen?

I am not entirely sure about setting a default value if it is `nil`; are we sure if it is `nil`, then the true running platform would be `linux/x86_64`?

Reply:
Yes, the SDK doesn't return that field if it's `LINUX/X86_64`. 
I think it's okay to set the default value knowing that this is the case. 🤔  Maybe there's some way to monitor if this ever changes.
I can imagine scenarios in which users would want to know/confirm that their container was built on the default platform, like if they're sometimes using WSL or changing the platform in their manifest or working on a team with different platforms.